internal/types/studies.go

```diff
@@ -6,9 +6,34 @@ import (
 
 type Study struct {
 	ModelAuditable
-	OwnerUserID uuid.UUID `gorm:"not null;index"`
-	Title       string    `gorm:"not null"`
-	Description string    `gorm:"type:text"`
+	OwnerUserID                      uuid.UUID `gorm:"not null;index"`
+	Title                            string    `gorm:"not null"`
+	Description                      *string   `gorm:"type:text"`
+	Admin                            *string   `gorm:"type:varchar(255)"`
+	Controller                       string    `gorm:"not null"`
+	ControllerOther                  *string   `gorm:"type:varchar(255)"`
+	InvolvesUclSponsorship           *bool     `gorm:""`
+	InvolvesCag                      *bool     `gorm:""`
+	CagReference                     *string   `gorm:"type:varchar(255)"`
```

Comment:
these and some of the other fields I think warrant inline comments to say what they are